XML/Other/ReplaceableTestParameters.xml

```diff
@@ -479,6 +479,10 @@ Scenario 2 : You need to run all the performance tests quickly.
 		<ReplaceThis>APACHE_CONCURRENCIES</ReplaceThis>
 		<ReplaceWith>(1 2 4 8 16 32 64 128 256 512 1024)</ReplaceWith>
 	</Parameter>
+	<Parameter>
+		<ReplaceThis>MARIADB_THREADS</ReplaceThis>
```

Comment:
Setting parameter as replaceableTestParameter enables the flexibility to override settings from parameter.

In the shell script perf_mariadb.sh, I set a default value if the parameter is not set, so that it can still run if we don't have this replaceableTestParameter.

Reply:
I am mocking Simon's last comment. If the values are not changed often, then we should add to the script directly.

Reply:
Yeah, the script can run directly, I set a default value if the parameter is not set. So the script doesn't depend on the replaceable test parameter.